types/steam-tradeoffer-manager/steam-tradeoffer-manager-tests.ts

```diff
@@ -152,3 +152,7 @@ manager.on('realTimeTradeConfirmationRequired', offer => {
         console.log(`Offer #${offer.id} requires confirmation`);
     }
 });
+
+const tradeOffer = manager.createOffer("12345678");
+
+console.log(tradeOffer.state);
```

Comment:
You can keep those tests, but it'd be important to add a test for the import using the path so we know that everything is exported as expected.

Reply:
Oh.